ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -553,6 +553,48 @@ export class WalletDB {
     }
   }
 
+  async *loadPendingTransactions(
+    account: Account,
+    headSequence: number,
+    tx?: IDatabaseTransaction,
+  ): AsyncGenerator<TransactionValue> {
+    const encoding = new PrefixEncoding(
+      BUFFER_ENCODING,
+      U32_ENCODING,
+      account.prefix.byteLength,
+    )
+
+    const noExpirationRange = StorageUtils.getPrefixesKeyRange(
```

Comment:
This is just supposed to be a single prefix right, not a start and end? We could use [`StorageUtils.getPrefixeKeyRange`](https://github.com/iron-fish/ironfish/blob/bc85577f559d4132857ba26328342e5e14c6b1c4/ironfish/src/storage/database/utils.ts#L16)?

Reply:
Yes! This should be just a single prefix